contributing/code/bc.rst

```diff
@@ -473,4 +473,69 @@ a return type is only possible with a child type.
 constructors of Attribute classes. Using PHP named arguments might break your
 code when upgrading to newer Symfony versions.
 
+Making Code Changes in a Backward Compatible Way
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As you read above, many changes are not allowed because they would represent a
+backward compability break. However, we want to be able to improve the code and
+its features over time and that can be done thanks to some strategies that
+allow to still do some unallowed changes in several steps that ensure backward
+compability and a smooth upgrade path. Some of them are described in the next
+sections.
+
+.. _add-argument-public-method:
+
+Adding an Argument to a Public Method
+.....................................
+
+Adding a new argument to a public method is possible only if this is the last
+argument of the method.
+
+If that's the case, here is how to do it properly in a minor version:
+
+#. Add the argument as a comment in the signature::
+
+        // the new argument can be optional
+        public function say(string $text, /* bool $stripWithespace = true */): void
+        {
+        }
+
+        // or required
+        public function say(string $text, /* bool $stripWithespace */): void
+        {
+        }
```

Comment:
Indention should be 4 spaces, instead of 8, no?

Reply:
done